### DISCUSSION TITLE
Add support for Inline Autofill (or password manager auto fill as a whole)

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#27041] [Ideas] [answered]
Hi there!
My I change my osu!account password recently and I need to re-login. The problem is that I'm using a password manager and I notice that when I'm trying to sign in to my account I expect a Gboard to show a list of my password like this
![Screenshot_20240205-122148.jpg](https://github.com/ppy/osu/assets/75104667/5930428e-301e-4382-b5ff-c19a11811ac0)

But I didn't get such prompt
https://github.com/ppy/osu/assets/75104667/36ae4ce5-4887-4f13-8036-a03fbf00937f

Are you planning to implement this useful feature? Cause it makes my life (And other people that use password manager) easier for them to sign in to osu!account on mobile phone.

**Comment by Rekunan** [comment, marked as answer]
Duplicate of #22565
